express/blocks/simplified-pricing-cards/simplified-pricing-tooltip.js

```diff
@@ -0,0 +1,52 @@
+import { getLibs } from '../../scripts/utils.js';
+import { getIconElementDeprecated } from '../../scripts/utils/icons.js';
+
+const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+
+export function adjustElementPosition() {
```

Comment:
why have this in a separate file? I'd just add it all to the simplified-pricing-cards.js so they load faster.

Reply:
Moved to the main block file.